FoodTracker/Controllers/MealTableViewController.swift

```diff
@@ -0,0 +1,163 @@
+//
+//  MealTableViewController.swift
+//  FoodTracker
+//
+//  Created by BrainX IOS 4 on 2020-12-14.
+//
+
+import UIKit
+import os.log
+
+class MealTableViewController: UIViewController {
+    
+    //MARK: -Properties
+    @IBOutlet var tableView: UITableView!
+    var meals = [Meal]()
+    
+    
+
+    override func viewDidLoad() {
```

Comment:
Every method and property should be under some mark.
These sort of methods viewDidLoad, viewWillAppear, viewDidAppear should be written `ViewController Lifecycle Methods`

Reply:
Done.